rust-ceramic/src/configuration.md

```diff
@@ -0,0 +1,10 @@
+# Configuration
+
+Configuration is done via environment variables. The following environment variables are available:
+
+- `CERAMIC_ONE_NETWORK`: Ceramic network (default: `testnet-clay`)
+- `CERAMIC_ONE_STORE_DIR`: Path to storage directory (default: `~/.ceramic-one`)
+- `CERAMIC_ONE_BIND_ADDRESS`: Bind address of the metrics endpoint (default: `5001`)
+- `CERAMIC_ONE_METRICS_BIND_ADDRESS`: Bind address of the metrics endpoint (default: `9464`)
+- `CERAMIC_ONE_SWARM_ADDRESSES`: Listen address of the p2p swarm (default: `/ip4/0.0.0.0/tcp/4001`)
+- `CERAMIC_ONE_LOCAL_NETWORK_ID`
```

Comment:
There are many more configuration env vars. All cli flags can be configured via the env. Should we highlight some important ones here and then refer to `ceramic-one daemon --help` for the rest?

Or if we could automate dumping the help test output here that would also work.

Reply:
Yeah, linked back to the help for the full list, just the important ones here makes sense to me.